posthog/session_recordings/test/__snapshots__/test_session_recordings.ambr

```diff
@@ -619,6 +619,27 @@
   '''
```

Comment:
Change in database.py caused the order of queries to change here. Should be fine